### PR TITLE
New Blog/News Post 

### PR DESCRIPTION
**Pull request by JamesQuinley**
Message from the team.

### CODE REVIEW

**Comment by JamesQuinley** [comment]
This won't serve to news page, working on that.

File: src/news/what-happened.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: What happened,and the next step.
+description: Let's talk about what happened, and what we're doing next
+date: 2022-20-10		
+tags:
+  - announcement
+---
+# What happened?
+A team restructure occured, that unfortunatly lead people to spread false information about the state of our metadata servers
```
Comment (Sneedplex) [inline src/news/what-happened.md:9]: unfortunately@

File: src/news/what-happened.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+title: What happened,and the next step.
```
Comment (Sneedplex) [inline src/news/what-happened.md:2]: space after comma

**Review by Sneedplex — CHANGES_REQUESTED** [review]

File: src/news/what-happened.md (hunk 1 above)
Comment (JamesQuinley) [reply src/news/what-happened.md:9]: mb fixing right now

File: src/news/what-happened.md (hunk 2 above)
Comment (JamesQuinley) [reply src/news/what-happened.md:2]: Done

**Comment by JamesQuinley** [comment]
This isn't showing the corrected spacing, will fix once home

File: src/news/what-happened.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+description: Let's talk about what happened, and what we're doing next
+date: 2022-20-10		
+tags:
+  - announcement
+---
+# What happened?
+A team restructure occured, that unfortunately lead people to spread false information about the state of our metadata servers
```
Comment (Sneedplex) [inline src/news/what-happened.md:9]: led, not lead

**Review by Sneedplex — CHANGES_REQUESTED** [review]

File: src/news/what-happened.md (hunk 3 above)
Comment (Sneedplex) [reply src/news/what-happened.md:9]: also, occurred, not occured

**Comment by JamesQuinley** [comment]
Seems the date was formatted incorrectly, that has been fixed and is ready for more review when you get back.

**Comment by JamesQuinley** [comment]
Sorry messed something up badly, retrying with a clean slate.
Won't make this mistake in the future.

**Pull request closed without merging**